src/npc.cpp

```diff
@@ -1348,8 +1353,13 @@ void npc::say( const std::string &line ) const
     if( g->u.sees( *this ) && g->u.is_deaf() ) {
         add_msg( m_warning, _( "%1$s says something but you can't hear it!" ), name );
     }
-    // Sound happens even if we can't hear it
-    sounds::sound( pos(), 16, sounds::sound_t::speech, sound );
+    // Hallucinations don't make noise when they speak
+    if( !is_hallucination() ) {
+        add_msg( _( "%1$s saying \"%2$s\"" ), name, formatted_line );
+        return;
+    }
+        // Sound happens even if we can't hear it
```

Comment:
```suggestion
    // Sound happens even if we can't hear it
```
this line and the next shouldn't be indented.